src/sentry/integrations/jira/integration.py

```diff
@@ -621,7 +621,7 @@ def sync_assignee_outbound(self, external_issue, user, assign=True, **kwargs):
                     continue
                 try:
                     jira_user = [
-                        r for r in res if r['emailAddress'] == ue.email
+                        r for r in res if r['emailAddress'].lower() == ue.email.lower()
```

Comment:
wondering if it's possible for `r['emailAddress']` to be None?

Reply:
Oh great point. It's not really clear to me that it couldn't be. I'll add another check there.